build/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
     dnf clean all
 
 ## Install yq in the image
-RUN curl -L https://github.com/mikefarah/yq/releases/download/v4.16.2/yq_linux_amd64 -o /usr/bin/yq &&\
+RUN curl -L https://github.com/mikefarah/yq/releases/download/v4.25.1/yq_linux_amd64 -o /usr/bin/yq &&\
```

Comment:
You can also use an ENV variable in the dockerfile to track the versions of the different packages

Reply:
yeah, I was unsure about using the latest or a fix version. I decided for a pinned version because this is a package fetched directly  from github, so I thought it would have been more safe to keep control over which version we fetch. I could change it anyway if you think is a better choice